sasdata/data_util/manipulations.py

```diff
@@ -282,8 +283,14 @@ class Binning:
     def __init__(self, min_value, max_value, n_bins, base=None):
         """
         if base is None: Linear binning
+        Todo: should base be a boolean? True = log? or do we want to allow for
```

Comment:
An Enum would probably be a better option to pre-define the binning types available. This would offer extensibility as well. Work for this is likely outside the scope of this PR.

Reply:
Have now updated that comment